service/src/test/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseServiceTest.java

```diff
@@ -495,6 +495,7 @@ void shouldRemoveHearingBookingWithSingleHearing() {
     class RemoveGatekeepingOrderUrgentHearingOrder {
 
         private final long caseId = 1L;
+        private final String fileName = "Test Filname.pdf";
```

Comment:
do we also need to update shouldNotThrowIfUrgentHearingOrderFound to use fileName